config/application.rb

```diff
@@ -60,7 +60,13 @@ class Application < Rails::Application
       end
     end
 
-    config.middleware.use Rack::Attack if Figaro.env.enable_rate_limiting == 'true'
+    if Figaro.env.enable_rate_limiting == 'true'
+      config.middleware.use Rack::Attack
```

Comment:
I guess technically this line is redundant now .... we could just only remove it if not needed?

Reply:
:shrug: Leaving it this way makes the code a little easier to understand.

Reply:
Yup, and it doesn't get double-included, so I'll just leave it